index.bs

```diff
@@ -3435,29 +3501,71 @@ partial dictionary MLOpSupportLimits {
         - *pow*: Compute the values of the values of the first input tensor to the power of the values of the second input tensor, element-wise.
 </div>
 
-<table id=tensor-limits-elementwise-binary class='data' link-for="MLGraphBuilder/add(a, b, options), MLGraphBuilder/sub(a, b, options), MLGraphBuilder/mul(a, b, options), MLGraphBuilder/div(a, b, options), MLGraphBuilder/max(a, b, options), MLGraphBuilder/min(a, b, options), MLGraphBuilder/pow(a, b, options)">
-  <caption>Tensor limits for element-wise binary options</caption>
+<table id=tensor-limits-elementwise-binary-part1 class='data' link-for="MLGraphBuilder/add(a, b, options), MLGraphBuilder/sub(a, b, options), MLGraphBuilder/mul(a, b, options), MLGraphBuilder/div(a, b, options), MLGraphBuilder/max(a, b, options), MLGraphBuilder/min(a, b, options)">
+  <caption>Tensor limits for {{MLGraphBuilder/add()}}/{{MLGraphBuilder/sub()}}/{{MLGraphBuilder/mul()}}/{{MLGraphBuilder/div()}}/{{MLGraphBuilder/max()}}/{{MLGraphBuilder/min()}}</caption>
   <thead>
     <tr>
       <th>operand</th>
       <th>[=/allowed data types=]</th>
+      <th>[=/minimum allowed data types=]</th>
       <th>[=/allowed ranks=]</th>
+      <th>[=/minimum allowed ranks=]</th>
     </tr>
   </thead>
   <tr>
     <td>{{a}}</td>
     <td>[=/any data type|any=]</td>
+    <td>{{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}, {{MLOperandDataType/"int32"}}</td>
     <td>[=/any rank|N=]</td>
+    <td>0 to 5</td>
   </tr>
   <tr>
     <td>{{b}}</td>
     <td>[=/same type as|same as=] {{a}}</td>
+    <td>{{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}, {{MLOperandDataType/"int32"}}</td>
     <td>[=/any rank|N=]</td>
+    <td>0 to 5</td>
   </tr>
   <tr>
     <td>*output*</td>
     <td>[=/same type as|same as=] {{a}}</td>
+    <td>{{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}, {{MLOperandDataType/"int32"}}</td>
     <td>[=/any rank|N=]</td>
+    <td>0 to 5</td>
+  </tr>
+</table>
+
+<table id==tensor-limits-elementwise-binary-part2 class='data' link-for="MLGraphBuilder/pow(a, b, options)">
+  <caption>Tensor limits for {{MLGraphBuilder/pow()}}</caption>
+  <thead>
+    <tr>
+      <th>operand</th>
+      <th>[=/allowed data types=]</th>
+      <th>[=/minimum allowed data types=]</th>
+      <th>[=/allowed ranks=]</th>
+      <th>[=/minimum allowed ranks=]</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>{{a}}</td>
+    <td>[=/any data type|any=]</td>
+    <td>{{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}, {{MLOperandDataType/"int32"}}</td>
+    <td>[=/any rank|N=]</td>
+    <td>0 to 4</td>
```

Comment:
Which ML API is the outlier for pow being 0 to 4 instead of 0 to 5? It will be problematic for callers if mul/div/pow have inconsistent ranks from each other, as certain substitution operations might expect that they can safely chain these or switch from one to another (e.g. mul(a,a) == pow(a,2)) without surprise. I propose the outlier API fix this in their Chromium backend with flanking reshapes (flatten to 1D if > 4D, pow, then back to the original shape).

Reply:
> Which ML API is the outlier for pow being 0 to 4 instead of 0 to 5? 

It's limited by TFLite backend. 
Please refer to https://source.chromium.org/chromium/chromium/src/+/main:services/webnn/tflite/graph_builder_tflite.cc;l=568

```c++
       // Limited to 4D when broadcasting is required:
       // https://source.chromium.org/chromium/chromium/src/+/main:third_party/tflite/src/tensorflow/lite/kernels/pow.cc
       /*pow_input=*/{kFloat16To32AndInt32, SupportedRanks::UpTo(4)},
```